source/install/trouble_mysql.rst

```diff
@@ -0,0 +1,282 @@
+
+The database created during installation is named ``mattermost``. If you
+fail to create this database or you misname it, when you attempt to
+start the Mattermost server you will see an error such as:
+
+ ::
+
+    [2017/09/20 17:11:37 EDT] [INFO] Pinging SQL master database
+    [2017/09/20 17:11:37 EDT] [EROR] Failed to ping DB retrying in 10 seconds
+    err-Error 1044: Access denied for user 'mmuser'@'%' to database 'mattermost'
+
+Note that MySQL is specifically denying access to the ``mattermost``
+database. This may mean that you have failed to create a database named
+``mattermost`` or you may have incorrectly referenced this database from
+the ``/opt/mattermost/config/config.json`` file.
+
+ **Checking that the Database Exists**
+
+To confirm that the ``mattermost`` database exists, open MySQL as root
+by executing ``mysql -u root -p``. When prompted, enter your
+password and then issue the command ``show databases;``. This command
+displays all the databases. You should see something similar to the
+following:
+
+ ::
+
+    +--------------------+
+    | Database           |
+    +--------------------+
+    | information_schema |
+    | mattermost         |
+    | mysql              |
+    | performance_schema |
+    | sys                |
+    +--------------------+
+    5 rows in set (0.03 sec)
+
+ **No ``mattermost`` Database**
+
+If the ``mattermost`` database doesn't exist, create a database named
+``mattermost`` by opening MySQL as root and issuing the command:
+``create database mattermost;``.
+
+If you accidentally created a database with the wrong name, you can
+remove it by issuing the command: :samp:`drop database {misnamed};`
+
+After creation of the database, attempt to restart the Mattermost server
+by navigating to the ``/opt/mattermost/bin`` directory and entering the
+command ``sudo -u mattermost ./platform``.
+
+**The mattermost Database Exists**
+
+If the ``mattermost`` database does exist, confirm that you have defined
+the database driver correctly in the
+``/opt/mattermost/config/config.json`` file. Open this file in a text
+editor, and review the value of ``"DataSource"``. It should be:
+
+ ::
+
+     "mmuser:*mmuser-password*@tcp(*host-name-or-IP*:3306)/mattermost?charset-utf8mb4,utf8&readTimeout-30s&writeTimeout-30s"
+
+You should also confirm that ``DriverName`` element (found immediately
+above the ``DataSource`` element) is set to ``mysql``.
+
+If you correct an error, restart the Mattermost server by navigating to
+the ``/opt/mattermost/bin`` directory and entering the command
+``sudo -u mattermost ./platform``.
+
+The Database User
+-----------------
+
+During installation you create a MySQL database user from the *mysql*
+prompt by issuing the command
+:samp: `create user 'mmuser'@'%' identified by '{mmuser-password}';`. The
+``mmuser-password`` value is a placeholder for the password you chose.
+You may also have specified an IP address rather than the wild card
+``%``.
+
+     **Note**
```

Comment:
```suggestion
.. note::
```

If you're not sure how special notes are formatted, you can take a look at how it's done on other docs, e.g. https://docs.mattermost.com/deployment/customize-email.html.